build/iOS/TestIOSApp/log_bridge.c

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "log_bridge.h"
+
+static struct _LogOutputCallbackInfo {
+    LogOutputCallback callback;
+    LogOutputContext context;
+} s_callbackInfo = {};
+
+void
```

Comment:
Function declaration should look different, please change according to code style.